doc/Migration.md

```diff
@@ -7,6 +7,13 @@ Please see the latest version (`master`) for the most up-to-date information.
 
 ## Guide
 
+### v1.17.0
+
+#### ES2017
+- Theia was updated to ES2017
+  - es5 VS Code extensions and Theia plugins are still supported
+  - If you require an es5 codebase you should be able to transpile back to es5 using webpack
```

Comment:
@sgraband I guess you researched how to do this. Do you have a link to some resource describing how to do the recompile?

Reply:
I updated the migration guide with information on how to transpile back.